platform/src/test/java/org/hillview/test/storage/SSTableTest.java

```diff
@@ -54,6 +54,7 @@ public CassandraConnectionInfo getConnectionInfo() {
         conn.user = "";
         conn.password = "";
         conn.lazyLoading = true;
+        conn.snapshotName = "myTestSnapshot";
```

Comment:
Do you have to add this to the UI as well?

Reply:
Yes, I will do some changes in the UI and let you know when it's ready